clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/CommitEvent.java

```diff
@@ -22,14 +22,14 @@
 import java.util.Collections;
 import java.util.Map;
 
-public abstract class CommitApplicationEvent extends CompletableApplicationEvent<Void> {
+public abstract class CommitEvent extends CompletableApplicationEvent<Void> {
 
     /**
      * Offsets to commit per partition.
      */
     private final Map<TopicPartition, OffsetAndMetadata> offsets;
 
-    public CommitApplicationEvent(final Map<TopicPartition, OffsetAndMetadata> offsets, Type type) {
+    protected CommitEvent(final Type type, final Map<TopicPartition, OffsetAndMetadata> offsets) {
```

Comment:
I reordered the parameters and made it `protected` since its an abstract class. Just being a little anal again.